x-pack/plugins/security_solution/cypress/tasks/alerts_detection_rules.ts

```diff
@@ -291,14 +301,16 @@ export const waitForRulesTableToBeRefreshed = () => {
 };
 
 export const waitForPrebuiltDetectionRulesToBeLoaded = () => {
-  cy.get(LOAD_PREBUILT_RULES_BTN).should('not.exist');
+  cy.log('Wait for prebuilt rules to be loaded');
+  cy.get(LOAD_PREBUILT_RULES_BTN, { timeout: 300000 }).should('not.exist');
   cy.get(RULES_TABLE).should('exist');
   cy.get(RULES_TABLE_REFRESH_INDICATOR).should('not.exist');
 };
 
-export const waitForRuleToChangeStatus = () => {
-  cy.get(RULE_SWITCH_LOADER).should('exist');
-  cy.get(RULE_SWITCH_LOADER).should('not.exist');
+export const waitForRuleToUpdate = () => {
+  cy.log('Wait for rules to update');
+  cy.get(RULE_SWITCH_LOADER, { timeout: 300000 }).should('exist');
+  cy.get(RULE_SWITCH_LOADER, { timeout: 300000 }).should('not.exist');
 };
```

Comment:
It would be great to add a comment for this function and clarify what it exactly does and where/when it can be used. Because the name is kinda abstract. Perhaps some renaming wouldn't hurt.